### DISCUSSION TITLE
Updated code for core.setoutput

### DISCUSSION THREAD

**Discussion by lalith-mcw** [community/community#38779] [Other Feature Feedback, Questions, & Ideas] [answered]
Question

Whats the updated syntax for core.setoutput since the gitactions are showing warning that set-output is depcrecated

**Comment by airtower-luna** [comment, marked as answer]
[According to the deprecation post](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#patching-your-actions-and-workflows) you only need to update `@actions/core` package to `v1.10.0` or later, and the function will use the new syntax.

**Reply by aguinaldok4** [reply]
include ':app'

**Comment by tabacostudio** [comment]
d5bae4f1711bea43baad5ec9544951fb2b0f1507

**Reply by mohon8156** [reply]
Love this

**Comment by chalupsky** [comment]
> [According to the deprecation post](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#patching-your-actions-and-workflows) you only need to update `@actions/core` package to `v1.10.0` or later, and the function will use the new syntax.

**Comment by Hisseinadamissa** [comment]
Hello

**Reply by sirjake6192** [reply]
Hey

**Comment by Biluyua** [comment]
Because

**Comment by F153** [comment]
Grazie e gooo

**Comment by Damon4277** [comment]
> Question
> 
> Whats the updated syntax for core.setoutput since the gitactions are showing warning that set-output is depcrecated

6 to 8

**Reply by Damon4277** [reply]
6 to 8

**Comment by Ng-group** [comment]
@Ng group uk ]()

**Comment by markalmeida26** [comment]
Treading platform

**Comment by Kocins** [comment]
Io

**Comment by phitattoo** [comment]
> Question
> 
> 
> 
> Whats the updated syntax for core.setoutput since the gitactions are showing warning that set-output is depcrecated

**Comment by 0985254122** [comment]
從我的iPhone傳送Kristian Aguinaldo ***@***.***> 於 2023年5月15日 下午9:50 寫道：﻿
include ':app'
